openedx/core/djangoapps/content/course_overviews/migrations/0015_courseoverview_license.py

```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
```

Comment:
We'll have to manually roll back this migration in stage.  I can do that once the pipeline for this is running.